src/hardware/mixer.cpp

```diff
@@ -3126,6 +3138,17 @@ static void init_mixer_dosbox_settings(Section_prop& sec_prop)
 	        "  - Use the MIXER command to fine-tune the chorus levels per channel.");
 	string_prop->Set_values({"off", "on", "light", "normal", "strong"});
 
+	bool_prop = sec_prop.Add_bool("denoiser", WhenIdle, DefaultOn);
+	bool_prop->Set_help(
+	        "Remove low-level residual noise from the output of the OPL synth and the Roland\n"
+	        "Sound Canvas. The emulation of these devices is accurate to the original\n"
+	        "hardware units, which includes the emulation of a very low-level semi-random\n"
+	        "noise. Although this is authentic, most people would find it slightly annoying.\n"
+	        "  off:  Disable the denoiser.\n"
+	        "  on:   Enable the denoiser on the OPL and SOUNDCANVAS channels (default).\n"
```

Comment:
Minor and not entirely unique to this config option: You've documented this as "on/off" but when you generate a new config it defaults to "true". true/false still works as expected though.

Reply:
Yeah I changed all descriptions to on/off from true/false a while ago. It's more user friendly. I'll look into writing the defaults as on and off instead.

Btw, 1, enabled, on, true are all valid "positive" options, then 0, none, off, disabled, and false are the "negative" variants.

Reply:
Fixed in https://github.com/dosbox-staging/dosbox-staging/pull/4248